components/InternalTable.module.css

```diff
@@ -0,0 +1,51 @@
+.tableContainer {
+	box-shadow: none;
+	background-color: var(--colour-lightGrey);
+	color: var(--colour-font);
+}
+
+.col1 {
+	width: 10%;
+	background-color: var(--table-left-edge);
+}
+
+.col2 {
+	width: 50%;
+	background-color: var(--colour-background);
+}
+
+.col3 {
+	width: 20%;
+	background-color: var(--colour-background);
+}
+
+.col4 {
+	width: 20%;
+	background-color: var(--colour-background);
+}
+
+.tableCellStyle {
+	font-weight: bold;
+	font-family: var(--primary-font-family);
+	font-size: 1.1rem;
+	background-color: #f5f5f5;
+	color: var(--colour-font);
+	margin-top: 1rem;
```

Comment:
I think there either shouldn't be a gap (like in the figma designs) or if there is then it should have rounded corners like the rest of the app. 
<img width="983" alt="image" src="https://user-images.githubusercontent.com/81656054/188255484-f85b8d74-c651-439f-b0e0-677532d53b9b.png">